ValidateCommitMessage.yml

```
@@ -7,37 +7,225 @@
+        echo "##vso[task.setvariable variable=packageVersion;isOutput=true]$extracted_version"
       else
-        echo "Error: The commit message does not contain a valid version format (versionX.X.X)."
+        echo "Error: The commit message does not contain a valid version format (version X.X.X)."
         echo "Last commit message: $last_commit_message"
         exit 1
       fi
-    displayName: "Check commit message format"
 
+      sed -i "s/version='[0-9]\+\.[0-9]\+\.[0-9]\+'$/version='$extracted_version'/" ./setup.py
+      sed -i "s/pythonrest_version = \"[0-9]\+\.[0-9]\+\.[0-9]\+\"/pythonrest_version = \"$extracted_version\"/" ./pythonrest.py
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)